rules-emerging-threats/2025/Exploits/CVE-2025-61882/web_cve_2025_61882_oracle_e_business_exploit.yml

```diff
@@ -0,0 +1,28 @@
+title: Oracle EBS CVE-2025-61882 Exploitation Attempt
+id: 7b31b049-c803-4a93-a0b1-d4a2c9aafb09
+status: experimental
+description: |
+    Detects potential exploitation attempts targeting Oracle E-Business Suite CVE-2025-61882 vulnerability in BI Publisher Integration.
+references:
+    - https://www.oracle.com/security-alerts/alert-cve-2025-61882.html
+    - https://labs.watchtowr.com/well-well-well-its-another-day-oracle-e-business-suite-pre-auth-rce-chain-cve-2025-61882well-well-well-its-another-day-oracle-e-business-suite-pre-auth-rce-chain-cve-2025-61882/
+    - https://www.crowdstrike.com/en-us/blog/crowdstrike-identifies-campaign-targeting-oracle-e-business-suite-zero-day-CVE-2025-61882/
+author: Saiprashanth Pulisetti ( @0xPrashanthSec)
+date: 2025-10-08
+tags:
+    - attack.initial-access
+    - attack.execution
+    - attack.t1190
+    - attack.t1203
+    - cve.2025-61882
+    - detection.emerging-threats
+logsource:
+    category: webserver
+detection:
+    selection:
+        cs-method: POST
+        cs-uri-stem: /OA_HTML/SyncServlet
+    condition: selection
```

Comment:
This strips down the rule too much. I highly suggest you read https://www.crowdstrike.com/en-us/blog/crowdstrike-identifies-campaign-targeting-oracle-e-business-suite-zero-day-CVE-2025-61882/

As well as play with https://github.com/watchtowrlabs/watchTowr-vs-Oracle-E-Business-Suite-CVE-2025-61882/ to generate sample logs.

I suggest you focus on the POST/GET requests that are required, the path traversal reported in the poc and in watchtower blog, and any GET params that increase the value

See also

<img width="1104" height="963" alt="Image" src="https://github.com/user-attachments/assets/518854b4-15cf-41b7-8890-17e00f16f23b" />

<img width="1101" height="2054" alt="Image" src="https://github.com/user-attachments/assets/95150ad8-6eaf-486f-b3e9-dab27ab7c0f0" />